includes/Ajax.php

```
@@ -98,32 +103,32 @@ public function hide_wedocs_rating() {
-            update_post_meta( $post_id, $type, $count + 1 );
+        if ($type) {
+            $count = (int) get_post_meta($post_id, $type, true);
+            update_post_meta($post_id, $type, $count + 1);
 
-            array_push( $previous, $post_id );
-            $cookie_val = implode( ',', $previous );
+            array_push($previous, $post_id);
+            $cookie_val = implode(',', $previous);
 
-            $val = setcookie( 'wedocs_response', $cookie_val, time() + WEEK_IN_SECONDS, COOKIEPATH, COOKIE_DOMAIN );
+            $val = setcookie('wedocs_response', $cookie_val, time() + WEEK_IN_SECONDS, COOKIEPATH, COOKIE_DOMAIN);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$val'. (undefined)

(UnusedLocalVariable)